chapters/synchronization.txt

```diff
@@ -3999,6 +3985,36 @@ ifdef::VK_NV_mesh_shader[]
     pname:dstStageMask must: not contain
     ename:VK_PIPELINE_STAGE_TASK_SHADER_BIT_NV
 endif::VK_NV_mesh_shader[]
+  * Each element of pname:pMemoryBarriers must: not have any access flag
+    included in its pname:srcAccessMask member that is not supported by any
+    of the pipeline stages in pname:srcStageMask, as specified in the
```

Comment:
Is the double negative necessary?

"must include only flags supported by pipeline stages"?

Reply:
True, was just mirroring existing language, but that would be easier.